### ISSUE TITLE
Add support for Debian 10

### ISSUE THREAD

**Issue by tvories** [samdoran/ansible-role-fish#1]
I was reading through the code and it appears you have hard-coded a .0 after `{{ ansible_facts.distribution_major_version }}` in the repo urls in vars/Debian.yml.  Here is the code you have in the vars directory:

```yaml
fish_apt_key_url: https://download.opensuse.org/repositories/shells:/fish:/release:/{{ fish_release_version }}/Debian_{{ ansible_facts.distribution_major_version }}.0/Release.key
fish_apt_repo_url: deb http://download.opensuse.org/repositories/shells:/fish:/release:/{{ fish_release_version }}/Debian_{{ ansible_facts.distribution_major_version }}.0/ /
```

It appears that the repo index stopped adding the .0 as of Debian 10 and it throws off the install.  See the screenshot below for my example.

![image](https://user-images.githubusercontent.com/13934524/69104620-28564980-0a26-11ea-9c51-a75016a094b1.png) 

I'm not sure how you could do this while avoiding hardcoding a separate option for Debian 10.  I'd be happy to submit a PR with the hardcode, but I wanted to see if you had a better idea first.

**Comment by tvories** [comment]
Submitted PR #2

**Comment by samdoran** [comment]
Resolved by #2
